tests/Fuzzed.nim

```diff
@@ -1,29 +1,52 @@
 import unittest
 
+#Util lib.
+import ../src/lib/Util
+
+#Random standard lib.
+import random
+
 const TEST_FUZZING_LOW {.intdefine.}: int = 1
 const TEST_FUZZING_MID {.intdefine.}: int = 4
 const TEST_FUZZING_HIGH {.intdefine.}: int = 8
 
+
+template setupRandom =
```

Comment:
Please use empty parentheses.

Reply:
Side note, very happy with how cleanly this was implemented. Good job :D I'd like to make sure it's noted this isn't thread safe, a concern you originally brought up. This is completely acceptable as we don't thread our tests as of right now (and can't, even with unittest2).